language-sql.md

```diff
@@ -3,11 +3,33 @@
 ### Minimum Requirements
 
 - MySQL keywords in uppercase
-- Column names in lowercase
+    E.g.: 
+    ```
+    UPDATE `consumer` SET `PASSWORD` = MD5(`PASSWORD`);
+    ```
+
+- Database table names must be plural. 
+    E.g.: 
+    `users`, `subscriptions`
+    
+    for Linking tables :
+    `users_subscriptions`
```

Comment:
Thanks @cpanganiban. Or maybe I should add to the description about the table relationships ?